src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
```

Comment:
Except on fairly rare occasion, my code never exceed 20-30 lloc, so that's not chocking for me :P